echo/frontend/src/components/form/MarkdownWYSIWYG/styles.css

```
@@ -1633,80 +1566,90 @@
   top: -6px;
   left: -6px;
   cursor: nw-resize;
 }
 
-._placeholder_uazmk_1084 {
+._imagePlaceholder_sects_1093 {
+  border: 2px dashed;
+  padding: 48px;
+  margin: 12px;
+  width: -moz-fit-content;
+  width: fit-content;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)